### PR TITLE
Prison food tastes gross now.

### PR DESCRIPTION
**Pull request by harmonyn**
:cl: bandit
tweak: Due to cost-cutting measures throughout the company, Nanotrasen has changed the recipes for food supplied to sustenance vendors in all station brigs. Nanotrasen prides itself in its quality standards and is sure crew members will notice no difference in taste!
/:cl:

>get thrown into perma
>eat some food out of the sustenance vendor explicitly stated to contain gross food
>"I love this taste!"

This just creates subtypes of food for sustenance vendors, with the GROSS attribute, so if you are a species that likes gross food (flypeople for instance) you will also like prison food.

### CODE REVIEW

File: code/modules/food_and_drinks/food/snacks_other.dm [hunk 1]
```diff
@@ -39,6 +39,14 @@
 	tastes = list("candy corn" = 1)
 	foodtype = JUNKFOOD | SUGAR
 
+/obj/item/reagent_containers/food/snacks/candy_corn/prison
+	name = "desiccated candy corn"
+	desc = "If this candy corn were any harder Security would confiscate it for being a potential shiv."
+	force = 1 // the description isn't lying
+	throwforce = 1 // if someone manages to bust out of jail with candy corn god bless them
```
Comment (Barhandar) [inline code/modules/food_and_drinks/food/snacks_other.dm]: :laughing:

**Comment by DaxDupont** [comment]
lizard buff

**Comment by Kagus** [comment]
> lizard buff

No, lizards don't go to jail.

Lizards go in the ground.

**Comment by bgobandit** [comment]
I considered making it a separate PRISON foodtype that all species hated but that seemed redundant

**Comment by Bawhoppen** [comment]
Having unique food rather than a subtype would be neat too.

**Comment by bgobandit** [comment]
was also strongly considering adding pruno as contraband

File: code/modules/food_and_drinks/food/snacks_other.dm (hunk 1 above)
Comment (Borisvanmemes) [inline code/modules/food_and_drinks/food/snacks_other.dm]: Drat, now i need to also get rid of the candy corn thats in perma before tossing someone in there!

**Comment by bgobandit** [comment]
so uh is there anything I need to do to fix this or

File: code/modules/food_and_drinks/drinks/drinks.dm [hunk 2]
```diff
@@ -207,6 +207,11 @@
 	spillable = TRUE
 	isGlass = FALSE
 
+/obj/item/reagent_containers/food/drinks/ice/prison
+	name = "dirty ice cup"
+	desc = "Either Nanotrasen's water supply is contaminated, or this machine actually vends lemon, chocolate, and cherry snow cones."
+	list_reagents  = list("ice" = 25, "liquidgibs" = 5)
```
Comment (duncathan) [inline code/modules/food_and_drinks/drinks/drinks.dm]: is the ice cup not supposed to be gross as well or no

**Pull request merged**